lvsfunc/scale.py

```diff
@@ -288,7 +292,7 @@ def descale(clip: vs.VideoNode,
 
 def ssim_downsample(clip: vs.VideoNode, width: int | None = None, height: int = 720,
                     smooth: float | VSFunction = ((3 ** 2 - 1) / 12) ** 0.5,
-                    kernel: Kernel | str = Catrom(), gamma: bool = False,
+                    kernel: Kernel | str = Bicubic(b=0, c=1/2), gamma: bool = False,
```

Comment:
Why did you change all Catrom occurrencies to Bicubic(b=0, c=0.5) or Bicubic(b=0, c=1/2)? Really xd